server/graphql/programs/src/queries/link_patient_to_store.rs

```diff
@@ -75,6 +75,8 @@ fn map_result(
         })),
         Err(err) => {
             let formatted_error = format!("{:#?}", err);
+            println!("Error while linking patient: {}", formatted_error);
```

Comment:
Added this so could see a bit more in the logs when I was getting errors, `ConnectionError` just returns a generic message to the frontend. Happy to leave?

Reply:
James mentioned a few times that our logging for api is not great, added issue: https://github.com/msupply-foundation/open-msupply/issues/7243

Reply:
Can you use log::error! here please

Reply:
log is not available in graphql layer... i can move to log from service call?